### DISCUSSION TITLE
What's missing for the version 3 release?

### DISCUSSION THREAD

**Discussion by ManuelRauber** [leoafarias/fvm#591] [General]
Hi @leoafarias,

is there an overview what's missing until you can release v3? 
Is there anything you need help with?

cheers!

**Comment by leoafarias** [comment]
Hi @ManuelRauber, the main thing pending is updating the documentation to reflect the changes to the commands and API, and the new way behavior for VsCode, and gitignore.

If you could help with it it would be amazing!

**Reply by ManuelRauber** [reply]
If there is something where I can be helpful, sure. 

I'm not familiar with the changes to commands and API. If you have some points I can look at and check the changes, I can try to adapt the documentation.

Regarding VSCode: I'm not using it, so I may be the wrong person to contribute to that.

If this is all that's missing, it would be nice to finish it soon, so you can release v3.

**Reply by leoafarias** [reply]
Yes, I'm aiming for this month! Thanks!
